### PR TITLE
access-control: change detection of SSH daemons

### PR DESCRIPTION
**Pull request by jmontesi**

### CODE REVIEW

File: cnf-certification-test/accesscontrol/suite.go [hunk 1]
```diff
@@ -562,45 +562,36 @@ func TestPodTolerationBypass(env *provider.TestEnvironment) {
 }
 
 const (
-	listProcessesCmd     = "ls -l /proc/*/exe"
-	sshDaemonProcessName = "sshd"
+	listProcessesCmd       = "ls -l /proc/*/exe"
```
Comment (rdavid) [inline cnf-certification-test/accesscontrol/suite.go:565]: You can remove unused constants,

File: cnf-certification-test/accesscontrol/suite.go [hunk 2]
```diff
@@ -562,45 +562,36 @@ func TestPodTolerationBypass(env *provider.TestEnvironment) {
...
+	sshDaemonProcessName   = "sshd"
+	sshServicePortNumber   = 22
+	sshServicePortProtocol = "TCP"
 )
 
 func TestNoSSHDaemonsAllowed(env *provider.TestEnvironment) {
-	var badContainers []string
-	var errorContainers []string
+	var badPods []string
+	var errorPods []string
```
Comment (rdavid) [inline cnf-certification-test/accesscontrol/suite.go:573]: `badPods` and `errPods` looks nicer.

**Review by rdavid — APPROVED** [review]

File: cnf-certification-test/accesscontrol/suite.go (hunk 1 above)
Comment (jmontesi) [reply cnf-certification-test/accesscontrol/suite.go]: Done.

File: cnf-certification-test/accesscontrol/suite.go (hunk 2 above)
Comment (jmontesi) [reply cnf-certification-test/accesscontrol/suite.go]: Done.

**Review by rdavid — APPROVED** [review]

**Review by greyerof — APPROVED** [review]

**Pull request merged**